app/controllers/change_requests_controller.rb

```diff
@@ -53,6 +64,82 @@ def create_change_request
     render status: :created, json: ChangeRequestsPresenter.present(@change_request)
   end
 
+  def  handle_insert_change_request
+    change_request = params[:change_request]
```

Comment:
Also can you work with @richardxia whether or not you want to keep the existing structure that nests the properties to change as "field_changes;" your method currently does not do that.

Reply:
I could go either way. The older PR for this feature used [`field_changes`](https://github.com/ShelterTechSF/askdarcel-api/pull/520/files#diff-d111eb9645f57db8b0f6869b5975b9a1c22a7366882e4464378cdbba74d35b74R80) for some reason, but it looks like none of the existing endpoints actually expect data to be sent that way, so maybe we don't need to use `field_changes`.

It's pretty easy for me to change the frontend code to work either way, so just let me know what you decide.

I _think_ the reason the old PR did it that way is because the one thing that was in `:change_request` but not in `:field_changes` was the `:action` param, which I think is still useful to have.

Reply:
Going ahead with changing the structure to use `fiend_changes` and `action` in the change request for both adding new addresses and phones to existing resources!